Packs/Palo_Alto_Networks_Enterprise_DLP/Integrations/Palo_Alto_Networks_Enterprise_DLP/Palo_Alto_Networks_Enterprise_DLP.py

```diff
@@ -76,11 +101,49 @@ def _get_dlp_api_call(self, url_suffix: str):
             count += 1
 
         if res.status_code < 200 or res.status_code >= 300:
-            raise DemistoException("Request to {} failed with status code {}".format(url_suffix, res.status_code))
+            print_debug_msg(f"Get request to {self._base_url}{url_suffix} failed with status code {res.status_code}")
+            raise DemistoException(f"Request to  {self._base_url}{url_suffix} failed with status code {res.status_code}")
 
         result_json = {} if res.status_code == 204 else res.json()
         return result_json, res.status_code
 
+    def _post_dlp_api_call(self, url_suffix: str, payload: Dict = None):
+        """
+        Makes a POST HTTP(s) call to the DLP API
+        Args:
+            url_suffix: URL suffix for dlp api call
+            payload: Optional JSON payload
+        """
+        count = 0
+
+        while count < MAX_ATTEMPTS:
+            res = self._http_request(
+                method='POST',
+                headers={'Authorization': f"Bearer {self.access_token}"},
+                url_suffix=url_suffix,
+                json_data=payload,
+                ok_codes=[200, 201, 204],
+                error_handler=self._handle_403_errors,
+                resp_type='response',
+                return_empty_response=True
+            )
+            if res.status_code != 403:
+                break
+            count += 1
+
+        if res.status_code < 200 or res.status_code >= 300:
+            print_debug_msg(f"Post request to {self._base_url}{url_suffix} failed with status code {res.status_code}")
+            raise DemistoException(f"Request to  {self._base_url}{url_suffix} failed with status code {res.status_code}")
```

Comment:
This code should be unreachable since you set the `ok_codes=[200, 201, 204]`.
Please remove this code.

Reply:
I have removed the unreachable lines.